docs/crvUSD/pegkeepers/regulator.md

```diff
@@ -0,0 +1,983 @@
+
+    Returns: current admin (`address`).
+
+    ??? quote "Source code"
+
+        === "PegKeeperRegulator.vy"
+
+            ```vyper
+            admin: public(address)
+            ```
+
+    === "Example"
+        ```shell
+        >>> PegKeeperRegulator.admin()
+        'todo'
+        ```
+
+
+### `set_admin`
+!!! description "`PegKeeperRegulator.set_admin(_admin: address):`"
+
+    !!!guard "Guarded Methods"
+        This function can only be called by the `admin` of the contract.
+
+    Function to set a new admin for the contract.
+
+    Emits: `SetAdmin`
+
+    | Input    | Type      | Description  |
+    | -------- | --------- | ------------ |
+    | `_admin` | `address` | New admin address. |
+
+    ??? quote "Source code"
+
+        === "PegKeeperRegulator.vy"
+
+            ```vyper
+            event SetAdmin:
+                admin: address
+
+            admin: public(address) 
+
+            @external
+            def set_admin(_admin: address):
+                # We are not doing commit / apply because the owner will be a voting DAO anyway
+                # which has vote delays
+                assert msg.sender == self.admin
+                self.admin = _admin
+                log SetAdmin(_admin)
+            ```
+
+    === "Example"
+        ```shell
+        >>> PegKeeperRegulator.set_admin()
+        todo
+        ```
+
+
+### `emergency_admin`
+!!! description "`PegKeeperRegulator.emergency_admin() -> address: view`"
+
+    Getter for the current emergency admin address of the Regulator contract. This address can only be changed by the `admin` by calling the `set_emergency_admin` function.
+
+    Returns: emergency admin (`address`).
+
+    ??? quote "Source code"
+
+        === "PegKeeperRegulator.vy"
+
+            ```vyper
+            emergency_admin: public(address)
+            ```
+
+    === "Example"
+        ```shell
+        >>> PegKeeperRegulator.emergency_admin()
+        'todo'
+        ```
+
+
+### `set_emergency_admin`
+!!! description "`PegKeeperRegulator.`"
+
+    Function to set a new emergency admin for the contract.
+
+    Emits: `SetEmergencyAdmin`
+
+    | Input    | Type      | Description  |
+    | -------- | --------- | ------------ |
+    | `_admin` | `address` | New emergency admin address. |
+
+    ??? quote "Source code"
+
+        === "PegKeeperRegulator.vy"
+
+            ```vyper
+            event SetEmergencyAdmin:
+                admin: address
+
+            emergency_admin: public(address)
+
+            @external
+            def set_emergency_admin(_admin: address):
+                assert msg.sender == self.admin
+                self.emergency_admin = _admin
+                log SetEmergencyAdmin(_admin)
+            ```
+
+    === "Example"
+        ```shell
+        >>> PegKeeperRegulator.set_emergency_admin()
+        todo
+        ```
+
+
+
+
+!!! description "`PegKeeperRegulator.`"
```

Comment:
function name in description with parameters and any return value.

Below the description with Returns and Emits also needs completing